common/oatbox/user/UserLanguageService.php

```diff
@@ -38,7 +38,11 @@ class UserLanguageService extends ConfigurableService implements UserLanguageSer
      */
     public function getDefaultLanguage()
     {
-        return DEFAULT_LANG;
+        if($this->hasOption(self::OPTION_DEFAULT_LANGUAGE)) {
+            $language = $this->getOption(self::OPTION_DEFAULT_LANGUAGE);
+        }
+
+        return $language ?? DEFAULT_LANG;
```

Comment:
```suggestion
        return $this->getOption(self::OPTION_DEFAULT_LANGUAGE, DEFAULT_LANG);
```

Reply:
Changed in fb83b41c70fce3419ef8b131f5fa30e6e0a806bb, thanks :)